core/lib/generators/solidus/install/install_generator.rb

```diff
@@ -148,6 +137,21 @@ def install_payment_method
       end
     end
 
+    def install_auth_plugin
+      # Doesn't need to add solidus_auth_devise if solidus_bolt is being installed as it already has the gem
```

Comment:
I don't think we should rely on a transitive dependency. Although it's unlikely, in the case of `solidus_bolt` stopping depending on `solidus_auth_devise`, we want the host application to keep using it.